engine/src/cython/initialize.cpp

```diff
@@ -177,7 +177,7 @@ void printLoggerHeader(const std::string pathLogger, const std::string nameLogge
         {"kernels_logger",      "ral_id|query_id|kernel_id|is_kernel|kernel_type"},
         {"kernels_edges_logger","ral_id|query_id|source|sink"},
         {"task_logger",         "time_started|ral_id|query_id|kernel_id|duration_decaching|duration_execution|input_num_rows|input_num_bytes"},
-        {"cache_events_logger", "ral_id|query_id|source|sink|num_rows|num_bytes|event_type|timestamp_begin|timestamp_end"},
+        {"cache_events_logger", "ral_id|query_id|source|cache_id|num_rows|num_bytes|event_type|timestamp_begin|timestamp_end"},
```

Comment:
please rename `source` to `message_id`
and make sure that its the CacheData message_id that we are setting in there. There are some places where that is what you are setting and some other places where you are putting something else. If a message_id is not readily available, please leave it blank

Reply:
Done